cfme/tests/infrastructure/test_host.py

```diff
@@ -87,6 +92,6 @@ def test_multiple_host_bad_creds(setup_provider, provider):
                            'validate_host': cfme_host["name"]})
 
     sel.click(credential_form.validate_multi_host)
-    flash.assert_message_match('Cannot complete login due to an incorrect user name or password.')
+    flash.assert_message_match(msgs[provider.type])
```

Comment:
@RonnyPfannschmidt, I'm not sure I got you. 
msgs dict has definitions only for two provider types.

so, this statement can be easily replaced with 
```
if provider.one_of(RHEVMProvider):
 msg = 'blabla'
elif provider.one_of(VMwareProvider):
 msg = 'blabla again'
```
btw, what is the reason for having only two provider types in that dict ? What about other providers ?